### DISCUSSION TITLE
Change the width of bookmarls windows

### DISCUSSION THREAD

**Discussion by Piattica** [sumatrapdfreader/sumatrapdf#2972] [General]
I need to maximize the document windows keeping opened the bookmarks windows.

Is it possible to set the width of bookmarks windows? If yes, how?

I also would like to reduce the size of the right arrow. Is it possible?

![image](https://user-images.githubusercontent.com/97229963/191291555-182ef2d5-38f3-4ccd-9796-5c9505d52771.png)

**Comment by tactilis** [comment]
> Change the width of bookmarks windows
Is it possible to set the width of bookmarks windows? If yes, how?


Just drag the divider between the bookmarks and the document:
 

https://user-images.githubusercontent.com/4060092/191323941-91e559a3-208b-45a0-b484-64228fc19f2c.mp4

  
Almost all applications allow you to do this.

**Reply by Piattica** [reply]
Thanks but it is obvious.

**Reply by tactilis** [reply]
Yes. Maybe so.

But it wasn't clear what your question was actually asking.

**Comment by GitHubRulesOK** [comment]
there are limits to sidebar and it's also proportional to just under 1/2 width of window 
so minimum was set to not be invisible/unworkable in some older cases

my range is SidebarDx = 150 to 911 based on 1840 Virtual screen width

unsure what you mean about right arrow the scrollbar is set by windows and only appears when needed.
![image](https://user-images.githubusercontent.com/12754913/191324802-2a239ba8-7e96-4a39-b53c-a4fbeb0bfa41.png)
![image](https://user-images.githubusercontent.com/12754913/191325131-14f9c70b-6562-42b2-a9dc-3ece710db798.png)

**Reply by Piattica** [reply]
"_unsure what you mean about right arrow the scrollbar is set by windows and only appears when needed_."
I mean if it is possible to reduce this space 
![image](https://user-images.githubusercontent.com/97229963/191591771-02de2b84-fbb3-49b6-acbc-cc54da7210bd.png)

**Reply by GitHubRulesOK** [reply]
look at lower left of my lowest picture above. That column is reserved by the levels 0 1 2 3 4 etc  the ethos of  ToC **short** cuts like 1.0, 1.1, 1.2  1.2.1 seems to have been Adobe aided and abetted rather than aids by abated.
